test/e2e_node/node_shutdown_linux_test.go

```diff
@@ -576,13 +576,16 @@ func getGracePeriodOverrideTestPod(name string, node string, gracePeriod int64,
 					Image:   busyboxImage,
 					Command: []string{"sh", "-c"},
 					Args: []string{`
-_term() {
-	echo "Caught SIGTERM signal!"
-	while true; do sleep 5; done
-}
-trap _term SIGTERM
-while true; do sleep 5; done
-`},
+					sleep 9999999 &
+					PID=$!
+					_term() {
+						echo "Caught SIGTERM signal!"
+						wait $PID
```

Comment:
shouldn't this be `kill $PID`?

Reply:
The previous content here was
```
_term() {
	echo "Caught SIGTERM signal!"
	while true; do sleep 5; done
}
```
In the function `_term`, the process is not aborted, so you should use `wait $PID`

Reply:
Can we leave a comment here that the reason we do `wait $PID` is because we want to ensure that the process does not exist (i.e. we block on SIGTERM?)

Reply:
> Can we leave a comment here that the reason we do `wait $PID` is because we want to ensure that the process does not exist (i.e. we block on SIGTERM?)

I have added comments and committed